charts/xapp-hello-world/values.yaml

```diff
@@ -49,10 +49,10 @@ xappFrameworkConfig:
 ### Define the xapp_endpoints.json
 xappEndpoints:
   REDIS_URL: "redis://{{ .Release.Name }}-xapp-redis.{{ .Release.Namespace }}:6379"
-  KAFKA_URL: "kafka://{{ .Values.global.kubeIp }}:31090"
-  API_URL: "http://{{ .Values.global.kubeIp }}:31315"
-  NATS_URL: "nats://{{ .Values.global.kubeIp }}:31000"
-  NATS_URL_5G: "nats://{{ .Values.global.kubeIp }}:31100"
+  KAFKA_URL: "kafka://drax-kafka:9095"
+  API_URL: "http://drax-dashboard:5000"
+  NATS_URL: "nats://drax-nats-headless:4222"
+  NATS_URL_5G: "nats://drax-nats-headless:4222"
```

Comment:
I don't really like hard-coding this `drax` release name, though it makes sense as a default with how the dependencies are installed... It's at least way better than relying on the kube ip and node ports!

Reply:
Probably this should be indicated as a breaking change actually? As we break the default behavior where we could just fill in the kube-ip (even for installation in an external kubernetes cluster). So maybe we should make everything completely independent of the kube-ip and then properly break it by removing `kubeIp` too...

Reply:
Regarding the first question: maybe I could set a draxReleaseName: drax in the file and then have the urls reference that e.g. KAFKA_URL: "kafka://{{ .Values.draxReleaseName }}-kafka:9095"

Reply:
> Probably this should be indicated as a breaking change actually? As we break the default behavior where we could just fill in the kube-ip (specifically for installation in an external kubernetes cluster). So maybe we should make everything completely independent of the kube-ip and then properly break it by removing `kubeIp` too...

Regarding the second question indeed that would be a breaking change but I wonder whether it was ever the intention to deploy this in an external cluster and point to a dRAX cluster. @ezeljkovic any inputs here?

If this is the case then I suppose we can keep on using the kubeIp for now. Loki got assigned port 31000 which traditionally was used for the nats4g. So we could just point NATS_URL to the same endpoint as NATS_URL_5G for now

Reply:
> > Probably this should be indicated as a breaking change actually? As we break the default behavior where we could just fill in the kube-ip (specifically for installation in an external kubernetes cluster). So maybe we should make everything completely independent of the kube-ip and then properly break it by removing `kubeIp` too...
> 
> Regarding the second question indeed that would be a breaking change but I wonder whether it was ever the intention to deploy this in an external cluster and point to a dRAX cluster. @ezeljkovic any inputs here?
> 
> If this is the case then I suppose we can keep on using the kubeIp for now. Loki got assigned port 31000 which traditionally was used for the nats4g. So we could just point NATS_URL to the same endpoint as NATS_URL_5G for now

I see that in the helm chart the only place where we use the kubeIp is actually in these endpoints. I am wondering whether we could conditionally check for the existence or not of the kubeIp value. If it exists, we use the {{ .Values.global.kubeIp }}:PORT. If it doesn't we use the service_name:port